tests/test_metadata.py

```diff
@@ -0,0 +1,46 @@
+from dmcontent.metadata import ContentMetadata
+from dmcontent.utils import TemplateField
+
+
+class TestContentMetadata(object):
+    def test_content_metadata_field(self):
```

Comment:
To be more explicit, you could call this `test_content_metadata___get_attr__`. Or something.

Reply:
Could also test that it raises an attribute error if not found.